compiler/optimizer/InductionVariable.cpp

```diff
@@ -689,7 +689,7 @@ int32_t TR_LoopStrider::detectCanonicalizedPredictableLoops(TR_Structure *loopSt
                   }
 
                ListIterator<TR::Block> bi(&exitBlocks);
-               vcount_t visitCount = comp()->incVisitCount();
+               visitCount = comp()->incVisitCount();
```

Comment:
yeah, that's my bad. i was looking at visitCount's in the file and somehow got myself onto the one at line 4310 which wasn't touched in this commit but looks very similar. Thanks, @andrewcraik .